synapseclient/table.py

```diff
@@ -299,6 +299,8 @@
 import json
 from builtins import zip
 from typing import List, Dict
+import pandas as pd
```

Comment:
Since `pandas` isn't a required package to install, please follow the same format as other functions that use pandas / numpy.

```
    test_import_pandas()
    import pandas as pd
```